hello.ahk

```diff
@@ -1422,98 +1301,212 @@ z - undo
-; Unfold Recursively (Ctrl+K Ctrl+]) unfolds the region at the cursor and all regions inside that region.
-; Fold All (Ctrl+K Ctrl+0) folds all regions in the editor.
-; Unfold All (Ctrl+K Ctrl+J) unfolds all regions in the editor.
-; Fold Level X (Ctrl+K Ctrl+2 for level 2) folds all regions of level X, except the region at the current cursor position.
-; Fold All Block Comments (Ctrl+K Ctrl+/) folds all regions that start with a block comment token.
-;Split editor: Control + \
-;Expland selection: Shift + Alt + Arrows
-;Select occurances of word: Control F2
-;Insert line above: Enter
-;Look up shortcuts: Control K + S
-;Go to line: Control G
-;Look up hotkey functions(Control Shift P)
-;Split window? (Control)
-
-  ; To do:
-  ; Advanced: scroll through the mouse wheel to change the side buttons(?)
-  ; Sending the shortcut for the compose feature (c)
-  ; Navigate to the "To:" field or image references as last resort
-
-; Tasklist:
-; -ip Start with lines 1-500 only... General refactoring and organization - figure out consistent convention for describing the code as well as....
-; Write down a working BM convention system here: -ip --inprogress, -f --fix, -d --done, -p1 - p2 -p3,  --nav, --find(problem), -x --currently testing
-  ; This is in Notion
-; Put all code I'm not using into an archive script
-; -p1 Get this code on to github while properly using git ignore
-; --done -p1 Make a hotkey to click in the vertical center of the text editor so I can focus after scroll with F14 + j/k
-; GUI menu for deck, and submenu for tag ALSO GUI menu for
-  ; Hotkey: Capslock H
-  ; Making the GUI activation hotkeys: Capslock 1, 4
-  ; First menu of decks I frequent: All, All Generative, Coding
-  ; Sub menu of tags: LS,
-; Find Vimston's way to scroll in SM
-
-
-; --done Use regular suspend instead of entire NiftyWindows Script
-; --done  -p1 Fix Socratic questions script - Maybe make the pause button more accessible. Soln: #!c
-; --done Fix F13 + Windows + Shift + Side arrow hotkeys
-; --done Figure out the bookmark shortcuts, navigation, and usage
-; --done Change the old OCR and test out new one(need to download file from github)
-; --done Get suspension working for the following scripts: OCRv2, Priorities, Socratic Questions
-; --done -f -p1 Suspension hotkey
-; --done Fix Alt + Shift + Arrow + F13 hotkey on VSC
+	mouseclick, left
+	sleep 200
+	mouseclick, left
+	sleep 1500
+	send {tab}
+	sleep 300
+	send Coding - Launch School RB189
+	send {Enter}
+	sleep 300
+	send {Enter}
+	return
+
++!v::
+	mouseclick, left
+	sleep 200
+	mouseclick, left
+	sleep 1500
+	send {tab}
+	sleep 300
+	send Coding - Launch School RB185
+	send {Enter}
+	sleep 300
+	send {Enter}
+	return
+
++!c::
+	mouseclick, left
+	sleep 200
+	mouseclick, left
+	sleep 1500
+	send {tab}
+	sleep 300
+	send Coding - Launch School RB175
+	send {Enter}
+	sleep 300
+	send {Enter}
+	return
+
+sendToVid(key, titlewin) {
```

Comment:
I moved every functions to the bottom of the script so that it will be more organizable. I recommend that you do this all the time.